dataload/import_to_elasticsearch.py

```diff
@@ -165,6 +166,10 @@ def import_to_elasticsearch(files, clean):
 
     time.sleep(1)
 
+    with open(os.path.join(current_dir, 'charity_names.json')) as fd:
```

Comment:
What happens if this file is missing? Is there anything here to ensure that the script that grabs charity numbers is called at least once during a deploy? Or is that one for @Bjwebb ?